crates/language/src/language.rs

```diff
@@ -254,7 +254,7 @@ pub trait Language {
                 "..." => GritMetaValue::Dots,
                 _ => {
                     let mut s = s.to_owned();
-                    s.insert_str(0, GRIT_METAVARIABLE_PREFIX);
+                    s.insert_str(0, self.metavariable_prefix());
```

Comment:
This is the bug that took me a while to find.

Reply:
yeah this would be the first language we make use of alternate prefixes, good catch